composables/useKeyboardKeys.ts

```diff
@@ -0,0 +1,15 @@
+import { useMagicKeys } from '@vueuse/core'
+
+export function onPressControlEnter(callback: () => void) {
+  const { Meta, Control, Enter } = useMagicKeys()
+
+  return watch([Meta, Control, Enter], () => {
+    if ((Meta.value || Control.value) && Enter.value) {
+      callback()
+    }
+  })
+}
+
+export function useKeyboardKeys({ onPressControlEnter: handlePressControlEnter }) {
+  handlePressControlEnter && onPressControlEnter(handlePressControlEnter)
+}
```

Comment:
```suggestion
function handlePressControlEnter(callback: () => void) {
  const { Meta, Control, Enter } = useMagicKeys()

  return watch([Meta, Control, Enter], () => {
    if ((Meta.value || Control.value) && Enter.value) {
      callback()
    }
  })
}

export function useKeyboardKeys({ onPressControlEnter }) {
  onPressControlEnter && handlePressControlEnter(onPressControlEnter)
}
```

Reply:
I'd rather have `onPressControlEnter` as a helper method like `onModalAnimation`

`useKeyboardKeys` is not what I'm using

Reply:
done f8b92a7


![CleanShot 2024-10-19 at 07 07 12](https://github.com/user-attachments/assets/cbb9c183-dfbb-47dd-8ee0-127a6db1e5e1)